src/threads.js

```diff
@@ -5922,7 +5922,7 @@ VariableFrame.prototype.silentFind = function (name) {
     answer the closest variable frame containing
     the specified variable. Otherwise return null.
 */
-    if (this.vars[name] !== undefined) {
+    if (this.vars[name] !== undefined && !(this.vars[name] instanceof Function)) {
```

Comment:
I guess the idea is that every entry in the ```vars``` dictionary has to be of type ```Variable```. So, we could easily check that, which might be safer and easier. Let me investigate....